OrbitClientData/include/OrbitClientData/ModuleData.h

```diff
@@ -29,36 +29,32 @@ class ModuleData final {
   [[nodiscard]] uint64_t file_size() const { return module_info_.file_size(); }
   [[nodiscard]] const std::string& build_id() const { return module_info_.build_id(); }
   [[nodiscard]] uint64_t load_bias() const { return module_info_.load_bias(); }
-  [[nodiscard]] bool is_loaded() const { return is_loaded_; }
+  [[nodiscard]] bool is_loaded() const;
 
   // relative_address here is the absolute address minus the address this module was loaded at by
-  // the process (module start address)
-  [[nodiscard]] orbit_client_protos::FunctionInfo* FindFunctionByRelativeAddress(
+  // the process (module base address)
+  [[nodiscard]] const orbit_client_protos::FunctionInfo* FindFunctionByRelativeAddress(
       uint64_t relative_address, bool is_exact) const;
-  // TODO(antonrohr): The module_start_address parameter should not be needed here, but it is
+  // TODO(antonrohr): The module_base_address parameter should not be needed here, but it is
   // because FunctionInfo still contains the field module_base_address. As soon as that field is
   // gone, remove the parameter here
-  void LoadSymbols(const orbit_grpc_protos::ModuleSymbols& module_symbols,
-                   uint64_t module_start_address);
+  void AddSymbols(const orbit_grpc_protos::ModuleSymbols& module_symbols,
+                  uint64_t module_base_address);
   // TODO(antonrohr): Remove this method as soon as FunctionInfo does not contain an absolute
   // address anymore
   void ClearSymbols();
-  [[nodiscard]] orbit_client_protos::FunctionInfo* FindFunctionFromHash(uint64_t hash) const {
-    return hash_to_function_map_.contains(hash) ? hash_to_function_map_.at(hash) : nullptr;
-  }
-
-  [[nodiscard]] std::vector<const orbit_client_protos::FunctionInfo*> GetFunctionList() const;
-  [[nodiscard]] absl::flat_hash_map<uint64_t, const orbit_client_protos::FunctionInfo*>
-  GetOrbitFunctions() const;
+  [[nodiscard]] orbit_client_protos::FunctionInfo* FindFunctionFromHash(uint64_t hash) const;
+  [[nodiscard]] std::vector<const orbit_client_protos::FunctionInfo*> GetFunctions() const;
```

Comment:
if you are not modifying the vectors, maybe just also make them const?

Reply:
done